src/components/Switch/Switch.tsx

```diff
@@ -10,89 +11,85 @@ export type ToggleProps = SwitchProps & {
     uncheckedIconType?: IconType;
 };
 
-type IconType =
-    | 'material'
-    | 'material-community'
-    | 'simple-line-icon'
-    | 'zocial'
-    | 'font-awesome'
-    | 'octicon'
-    | 'ionicon'
-    | 'feather'
-    | 'fontisto'
-    | 'foundation'
-    | 'evilicon'
-    | 'entypo'
-    | 'antdesign'
-    | 'font-awesome-5'
-    | undefined;
+// TODO: Ask alex to create design tokens
+const MOTION_OVERSHOOT = Easing.bezier(0.175, 0.885, 0.32, 1.275);
 
 const Switch = ({
     trackColor,
     size = 32,
     thumbColor,
     onValueChange,
     disabled,
-    value,
+    value: valueProp,
     checkedIcon,
     unCheckedIcon,
     style,
     checkedIconType,
     uncheckedIconType,
```

Comment:
We need more props here to make it more customizable.

Reply:
Can you add thumbStyle, thumbContainerStyle, overlayStyle and iconStyle as props?

Reply:
done @thetaungg